app/lib/submission_builder/return_header.rb

```diff
@@ -26,6 +26,9 @@ def document
             end
             xml.TaxpayerSSN @submission.data_source.primary.ssn if @submission.data_source.primary.ssn.present?
             xml.DateOfBirth date_type(@submission.data_source.primary.birth_date) if @submission.data_source.primary.birth_date.present?
+            if @submission.data_source.state_code == "nc" && @submission.data_source.direct_file_data.phone_number.present?
```

Comment:
I'd really rather not do this. preferably we have the same return header for all states, but if not, I like your idea that we split out the states that are different to implement their own version